src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (40 * days) - 50;
+  } else if (days >= 3) {
+    return (40 * days) - 20;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item. You should create variables for main values - 20, 50, 40 they shouldn't be just magic numbers

Reply:
Also, checklist 3rd item, why do you have else if after a return statement? Code won't execute past this, use plain if construction